package.json

```diff
@@ -60,7 +60,8 @@
     "webpack-cli": "^6.0.1"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "4.8.4"
+    "@eppo/js-client-sdk-common": "4.8.4",
+    "spark-md5": "^3.0.2"
```

Comment:
Temp workaround until [js-common#207](https://github.com/Eppo-exp/js-sdk-common/pull/207) lands and is released